aspnetcore/mvc/razor-pages/index.md

```diff
@@ -27,20 +25,48 @@ If you're looking for a tutorial that uses the Model-View-Controller approach, s
 
 [!INCLUDE[install 2.0](../../includes/install2.0.md)]
 
+<a name="rpvs17"></a>
+
+## Creating a Razor Pages project
+
+Select the **.NET Core CLI** tab if you are working with:
+
+* [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/)
+* [Visual Studio Code](https://code.visualstudio.com/)
+* The CLI
+
+# [Visual Studio](#tab/vs0)
+
+To create a Razor Pages starter project:
+
+* From the Visual Studio **File** menu, select **New > Project**.
+* Create a new **ASP.NET Core Web Application**.
+ ![new ASP.NET Core Web Application](index/_static/npp.png)
+* Select **ASP.NET Core 2.0** in the dropdown, and then select **Web Application**.
+ ![Web Application (Razor Pages)](index/_static/npp2.png)
+
+[Getting started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start) contains detailed instructions on how to create a Razor Pages project using Visual Studio.
```

Comment:
Then why duplicate that info here?  Why not just link to it, given this is an intro and not a tutorial?